docs/src/main/sphinx/connector/faker.md

```diff
@@ -0,0 +1,225 @@
+# Faker connector
+
+The Faker connector generates random data matching a defined structure. It uses
+the [Datafaker](https://www.datafaker.net/) library, which makes the generated
+data more realistic. It can be used to easily populate another data source with
+large and realistic test data, which can help with testing performance of
+applications processing data (like Trino itself), and also application's user
+interface.
+
+## Usage
```

Comment:
I think we should move this section down after SQL support so we are consistent with other connectors.

Reply:
It makes me sad, but I moved it.

Reply:
Ultimately we should decide consistently for all connectors. We should have some of the generic stuff in some generic catalog docs / tutorial and then pull out redundant stuff from connector docs. I just never got around to completing that .. ideally @jhlodin @Ordinant and team could do more on that but I dont want to start this now in this connector without a larger plan .. I hope that makes sense